src/core/Song.cpp

```diff
@@ -1416,19 +1417,18 @@ void Song::exportProject( bool multiExport )
 				// Get first extension from selected dropdown.
 				// i.e. ".wav" from "WAV-File (*.wav), Dummy-File (*.dum)"
 				suffix = efd.selectedNameFilter().mid( stx + 2, etx - stx - 2 ).split( " " )[0].trimmed();
+				exportFileName.remove( "." + suffix );
```

Comment:
Two observations about `remove(...)`...

1.  By defaul it's case insensitive by default which can cause issues on Mac and Windows, since they're both case-insensitive (yes, Mac is too!)
2.  This will remove all instances of e.g. `.wav`, so there are some edge cases where `my.wav.file.wav` might get snagged, no?